### PR TITLE
Remove private def

### PR DESCRIPTION
**Pull request by sakshi-keepworks**
Bulk change to remove `private def` from everywhere.

https://github.com/rails/rails/commit/6fd9298272518f4a25a1494b1b1c860bda2dd6b4

### CODE REVIEW

**Comment by gameover7** [comment]
Why did you remove private from in front of def and simply move it above?

File: actioncable/test/channel/stream_test.rb [hunk 1]
```diff
@@ -26,7 +26,9 @@ def send_confirmation
       transmit_subscription_confirmation
     end
 
-    private def pick_coder(coder)
+    private
+
+    def pick_coder(coder)
```
Comment (gsamokovarov) [inline actioncable/test/channel/stream_test.rb]: Rails' style indents the methods after `private`.

Comment (sjain1107) [reply actioncable/test/channel/stream_test.rb]: Updated 👍

**Comment by kaspth** [comment]
@sjain1107 thanks!

@gameover7 see the referenced commit: 6fd9298

**Pull request merged**